plugins/inputs/diskio/README.md

```diff
@@ -19,6 +19,9 @@ The diskio input plugin gathers metrics about disk traffic and timing.
   ## Currently only Linux is supported via udev properties. You can view
   ## available properties for a device by running:
   ## 'udevadm info -q property -n /dev/sda'
+  ## Note: Most, but not all, udev properties can be accessed this way. Properties
+  ## that are currently accessible include the "IS_*" series and the "DEVLINKS" 
```

Comment:
Should be able to access any of the `E:` values, so that would be at least `ID*` and `DM*` and probably others.  Might be easier to document fields that cannot be looked up: DEVTYPE, DEVNAME, DEVPATH.

Reply:
Makes sense.